model_compression_toolkit/qat/keras/quantizer/quantization_dispatcher_builder.py

```diff
@@ -32,6 +34,40 @@
                                             qi.QuantizationMethod.POWER_OF_TWO: STEActivationQuantizer,
                                             qi.QuantizationMethod.UNIFORM: STEUniformActivationQuantizer}}
 
+def get_trainable_quantizer_weights_config(n: common.BaseNode):
+    """
+    Returns the relevant configurations for weights trainable quantizer
+
+    Args: None
+
+    Returns: TrainableQuantizerWeightsConfig object.
+
+    """
+    config = n.final_weights_quantization_cfg
+    return TrainableQuantizerWeightsConfig(config.weights_quantization_method,
```

Comment:
1. what's the point of this function if it only initializes the config class?
2. why not pass the config to the TrainableConfig and let its init parse it?

Reply:
this function is planned to be part of BaseNode functions. currently we decided to wait until we remove the dependency of quantization infrastructure in MCT. I will explain...